lib/opensusebasetest.pm

```diff
@@ -231,17 +231,19 @@ done", "binaries-with-missing-libraries.txt", {timeout => 60, noupload => 1});
 
 =head2 investigate_yast2_failure
 
- investigate_yast2_failure();
+ investigate_yast2_failure(logs_path => $logs_path);
 
-Inspect the YaST2 logfile checking for known issues.
+Inspect the YaST2 logfile checking for known issues. logs_path can be a directory where logs are saved
+e.g. /tmp. In that case the function will parse /tmp/var/log/YaST2/y2logs* files.
```

Comment:
not always will be with * according to the code.

Reply:
Yes, this is just an example as "e.g." suggests in the beginning of the sentence.